src/serverless/functions/recordingrules.js

```diff
@@ -0,0 +1,51 @@
+/* global Twilio */
+'use strict';
+
+// We need to use a newer twilio client than the one provided by context.getTwilioClient(),
```

Comment:
Does this mean that different parts of this project use different Twilio clients? Can we set something up to use the same Twilio client everywhere?

Reply:
> Does this mean that different parts of this project use different Twilio clients?

Yes, but I think that this will always be the case. Even if we set up all functions to use the same version of the Twilio Client, they will all still have to make their own instances of the client. 

For now, I'd like the `token` function to continue to obtain it's Twilio client via `context.getTwilioClient();`. Once Twilio Serverless upgrades (see: https://issues.corp.twilio.com/browse/RUN-3731) the version of the Twilio Client returned by `context.getTwilioClient();`, then we can change this file so that it also uses `context.getTwilioClient();` (instead of creating its own client).

Reply:
Okay sounds good. Add a comment with a `TODO` explaining this context.